zebra-chain/src/transparent/tests/vectors.rs

```diff
@@ -0,0 +1,39 @@
+use super::super::serialize::parse_coinbase_height;
+
+#[test]
+fn parse_coinbase_height_mins() {
+    zebra_test::init();
+
+    // examples with height 1:
+
+    let case1 = vec![0x51];
+    assert!(!parse_coinbase_height(case1.clone()).is_err());
+    assert_eq!(parse_coinbase_height(case1).unwrap().0 .0, 1);
```

Comment:
Nitpick: Extra space, possibly confusing

```suggestion
    assert_eq!(parse_coinbase_height(case1).unwrap().0.0, 1);
```

Reply:
Actually for some reason rustfmt is doing this :(

Reply:
Oh wow, yeah, then let's just leave it.

Maybe it's worth reporting a rustfmt bug, but that's totally optional:
https://github.com/rust-lang/rustfmt/issues/new

Reply:
I opened an issue https://github.com/rust-lang/rustfmt/issues/5123

Not sure if it is a bug, i think there should be a reason. I think that if you have to do something as `var.0.0` it should mean that the involved function and types are not the best way designed, however i do not agree with rustfmt on adding an extra space to gain some clarity initially.